core/chains/evm/client/chain_client.go

```diff
@@ -203,12 +256,20 @@ func (c *chainClient) PendingCodeAt(ctx context.Context, account common.Address)
 
 // TODO-1663: change this to evmtypes.Nonce(int64) once client.go is deprecated.
 func (c *chainClient) PendingNonceAt(ctx context.Context, account common.Address) (uint64, error) {
-	n, err := c.multiNode.PendingSequenceAt(ctx, account)
+	rpc, err := c.multiNode.SelectRPC()
+	if err != nil {
+		return 0, err
+	}
+	n, err := rpc.PendingSequenceAt(ctx, account)
 	return uint64(n), err
 }
 
 func (c *chainClient) SendTransaction(ctx context.Context, tx *types.Transaction) error {
-	return c.multiNode.SendTransaction(ctx, tx)
+	rpc, err := c.multiNode.SelectRPC()
+	if err != nil {
+		return err
+	}
+	return rpc.SendTransaction(ctx, tx)
```

Comment:
To implement SendTransaction we need to use TransactionSender component (that we need to create first).
TransactionSender component should contain MultiNode's logic for transaction sending.
Also it might be possible to merge PR with TransactionSender into develop before finishing this work

Reply:
Going to implement TransactionSender in a separate PR

Reply:
Wait, does it mean that till you implement TransactionSender, the current MultiNode code for SendTransaction will be deleted?

Reply:
Yes, but the TransactionSender is being implemented in another PR. Both will be merged to a base branch for soak testing before develop.